src/imageproxy.rs

```diff
@@ -175,6 +179,9 @@ impl TryFrom<ImageProxyConfig> for Command {
         } else if config.auth_anonymous {
             c.arg("--no-creds");
         }
+        if let Some(certificate_authority) = config.certificate_authority.as_deref() {
+            c.args(&["--src-cert-dir", certificate_authority]);
```

Comment:
Also, now that I look this should be `--cert-dir`; see `skopeo experimental-image-proxy --help`.  The `--src-cert-dir` is only for `skopeo copy` because it supports copying between two remote registries which may have disjoint certificates.